### PR TITLE
21.1.5 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: CS/DockPanelInCode_Ex/Window1.xaml [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
-                <dxdo:FloatGroup FloatSize="200,150" FloatLocation="100,50">
+                <dxdo:FloatGroup FloatLocation="100,50" FloatSize="200,150">
                     <dxdo:LayoutPanel x:Name="paneProperties" Caption="Properties">
                         <DockPanel>
-                            <Button Content="Dock making a split container" DockPanel.Dock="Top" Height="40" Click="Button1_Click" />
-                            <Button Content="Dock making a tab container" DockPanel.Dock="Top" Height="40" Click="Button2_Click" />
+                            <Button
+                                Height="40"
+                                Click="Button1_Click"
+                                Content="Dock making a split container"
```
Comment (albertov05) [inline CS/DockPanelInCode_Ex/Window1.xaml]: Is this string editable? If so, what do you mean by "making" in this context?

File: CS/DockPanelInCode_Ex/Window1.xaml [hunk 2]
```diff
@@ -1,31 +1,48 @@
...
+                                DockPanel.Dock="Top" />
+                            <Button
+                                Height="40"
+                                Click="Button2_Click"
+                                Content="Dock making a tab container"
```
Comment (albertov05) [inline CS/DockPanelInCode_Ex/Window1.xaml]: See my comment on line 35 above.

File: Readme.md [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
+- [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup)
+- [LayoutPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel)
+- [DockLayoutManager.DockController](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockController)
+
+## More Examples
+
+- [WPF Dock Layout Manager - Create a Simple Layout of Dock Panes](https://github.com/DevExpress-Examples/how-to-create-a-simple-layout-of-dock-panes-e1600)
+- [WPF Dock Layout Manager - Create a Complex Layout of Dock Panels](https://github.com/DevExpress-Examples/how-to-create-a-complex-layout-of-dock-panels-e1663)
+- [WPF Dock Layout Manager - Сreate a DocumentGroup with Two Tabs](https://github.com/DevExpress-Examples/how-to-create-a-documentgroup-with-two-tabs-e1670)
+- [WPF Dock Layout Manager - Create a Tabbed and Document Groups](https://github.com/DevExpress-Examples/how-to-create-a-tabbedgroup-and-documentgroup-groups-e1656)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
- [WPF Dock Layout Manager - Create Tabbed and Document Groups](https://github.com/DevExpress-Examples/how-to-create-a-tabbedgroup-and-documentgroup-groups-e1656)
```

File: CS/DockPanelInCode_Ex/Window1.xaml (hunk 1 above)
Comment (friedcucumber) [reply CS/DockPanelInCode_Ex/Window1.xaml]:
```suggestion
                                Content="Dock a panel to a split container"
```

File: CS/DockPanelInCode_Ex/Window1.xaml (hunk 2 above)
Comment (friedcucumber) [reply CS/DockPanelInCode_Ex/Window1.xaml]:
```suggestion
                                Content="Dock a panel to a tab container"
```

**Pull request merged**